Formula/pgcli.rb

```diff
@@ -92,12 +96,18 @@ class Pgcli < Formula
   def install
     venv = virtualenv_create(libexec, "python3.10")
 
+    resource("psycopg").stage do
+      inreplace "psycopg/pq/_pq_ctypes.py", "libname = find_libpq_full_path()",
+                                            "libname = '#{Formula["libpq"].opt_lib/shared_library("libpq")}'"
```

Comment:
I think this should be using `pg_config` to find `libpq` automatically -- is that not working?

Reply:
It does work when `pg_config` is on `PATH`, but `libpq` is keg only

Reply:
Right, we should add a comment documenting this. I think we may make `libpq` not-keg-only.

Reply:
Good idea, #111006